docker/importer/importer.py

```diff
@@ -174,10 +174,6 @@ def import_new_oss_fuzz_entries(self, repo, oss_fuzz_source):
 
   def schedule_regular_updates(self, repo, source_repo):
     """Schedule regular updates."""
-    if (source_repo.last_update_date and
```

Comment:
Sorry, I completely forgot about this case where we still need it. Let's revert to your old way for now.

Reply:
The condition we want to delete is the one in  `_process_updates_bucket`. This one is the the one we need to keep.

Reply:
Ah ok, should be fixed now